### PR TITLE
Printdb

### PR DESCRIPTION
**Pull request by IdanTo1**

### CODE REVIEW

File: printdb.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+def print_employees_report(cursor):
+    print("Employees report")
+    cursor.execute("""SELECT emp.name, emp.salary, cs.location, sum(prod.price * act.quantity)
+    FROM ((Employees AS emp
+    INNER JOIN Coffee_stands AS cs 
+    ON emp.coffee_stand = cs.id)
+    LEFT OUTER JOIN (Activities AS act
+    INNER JOIN Products AS prod ON act.product_id = prod.id)
```
Comment (Hroy02) [inline printdb.py:15]: Shouldn't the left outer join be replaced with inner join to remove cases of employees with no activities?

File: printdb.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    for one in all_data:
+        print(*one)
+
+
+# , sum(prod.price * act.quantity)
+# LEFT OUTER JOIN (Activities AS act
+#     INNER JOIN Products as prod on act.product_id = prod.id)
+#     ON emp.id = act.activator_id)
+#   ON emp.id = act.activator_id
+#   ORDER BY emp.name ASC
```
Comment (Hroy02) [inline printdb.py]: Commented out code should probably be removed

**Review by Hroy02 — CHANGES_REQUESTED** [review]
All in all looks good, note the comments

File: printdb.py (hunk 1 above)
Comment (IdanTo1) [reply printdb.py:15]: no. we want to print them as well. notice the new commit with the ifnull()
to make sure it says 0 and not None

File: printdb.py (hunk 2 above)
Comment (IdanTo1) [reply printdb.py]: removed before even seeing this comment :)

**Review by Hroy02 — APPROVED** [review]
Looks good!

**Pull request merged**